### ISSUE TITLE
Support for Tag-Based Searches (e.g., K-Pop, Dubstep, etc.)

### ISSUE THREAD

**Issue by DwieDima** [bsq-ports/BetterSongSearchQuest#28]
It would be great if the search functionality could include support for tags like "K-Pop," "Dubstep," etc. Currently, it's hard to narrow down by genre or style, and adding tag-based searches would really help filter results. For example, being able to type "K-Pop" and pull up all songs tagged with it would make things more efficient.

This could work like a genre filter that lets users quickly find songs in specific categories. Would love to see this kind of feature added! Thanks for all the great work on BetterSongSearch.

![image](https://github.com/user-attachments/assets/cc5439bc-0e2f-4036-a134-0f55e6999e7e)

**Comment by FrozenAlex** [comment]
Yeah it is planned, it's just I don't have time to implement it. PC version already has all the needed data in the dataset. We just need to update songdetails
